src/full/Agda/Compiler/MAlonzo/Primitives.hs

```diff
@@ -263,6 +263,7 @@ primBody s = maybe unimplemented (fromRight (hsVarUQ . HS.Ident) <$>) $
   ]
   where
   x |-> s = (x, Left <$> s)
+  -- TODO:: Defined but not used: ‘bin’
```

Comment:
If you want, comment it out, but please don't delete, might be used in the future.

Reply:
OK